plugins/models/resnet18.py

```diff
@@ -0,0 +1,67 @@
+import keras
+import keras_resnet
+import keras_resnet.models
+import tensorflow as tf
+
+from deepprofiler.learning.model import DeepProfilerModel
+
```

Comment:
We can add a link to the ResNet paper: https://www.cv-foundation.org/openaccess/content_cvpr_2016/papers/He_Deep_Residual_Learning_CVPR_2016_paper.pdf